src/_includes/banner.html

```diff
@@ -3,11 +3,7 @@
 {% endcomment -%}
 <div class="site-banner site-banner--default" role="alert">
   Flutter is back at Google I/O on May 10th!
```

Comment:
Can we have just one banner? It looks a bit confusing to both be promoting I/O *and* this change...